### DISCUSSION TITLE
fail2ban fails to ban Dovecot

### DISCUSSION THREAD

**Discussion by dkoski55** [fail2ban/fail2ban#3270] [General]
I have Debian 11 with nftables and fail2ban which works perfectly with ssh.  The server has Postfix and Dovecot and the standard config files for these did not seem to work.  I added custom-dovecot.conf which was created from a copy of dovecot.conf changing only the failregex parameter:
```
^conn unix:auth-worker \(pid=[0-9][0-9]*,uid=[0-9][0-9]*\): auth-worker<[0-9][0-9]*>: sql\(.*,<HOST>(,<[0-9A-Za-z][0-9A-Za-z]*>)?\): ((P|p)assword mismatch|(U|u)nknown user)
```
It works perfectly with fail2ban-regex:
```
fail2ban-regex --print-all-matched /var/log/mail.log /etc/fail2ban/filter.d/custom-dovecot.conf
```
I can see 12 matches from a test well within a minute.  But no action is taken.  Note that the default logging mode for Dovecot appears to consolidate individual failures into one error, thus the reason for changing Dovecot to verbose logging.
```ini
[custom-dovecot]
mode=agressive
enabled = true
port    = pop3,pop3s,imap,imaps,submission,465,sieve
filter  = dovecot
logpath  = /var/log/mail.log
maxretry = 4 
#findtime = 600
findtime = 6000
bantime = 1800
```
Thanks for your input.
David Koski

**Comment by sebres** [comment]
> I added custom-dovecot.conf which was created from a copy of dovecot.conf changing only the failregex parameter

You did not provide an excerpt with the example messages your RE must match, thus I don't know what exactly is/was incorrect and cannot suggest any expression here (your RE seems to be a bit vulnerable due to catch-all `.*` before `<HOST>`).

> I can see 12 matches from a test well within a minute. 

All 12 messages from the same IP? Related to your settings it must ban an IP after 4 attempts in 100 minutes.
Did you reload the fail2ban after the change of configs?
Also note that a restart/reload doesn't reparse the log from start (it will find new messages only).
